modules/ROOT/pages/migration-from-4x.adoc

```diff
@@ -874,20 +900,20 @@ These features have either changed or have been deleted in TinyMCE 5.
 | *Plugin name* | *Description*
 
 | link:{url}/docs-4x/plugins/contextmenu/[ContextMenu]
-| Context menus are now part of the {productname} core. For information on adding custom context menus, see: link:{baseurl}/ui-components/contextmenu/[UI components - Context menu].
+| Context menus are now part of the {productname} core. For information on adding custom context menus, see: xref:ui-components/contextmenu.adoc[UI components - Context menu].
 
 | link:{url}/docs-4x/plugins/textcolor/[TextColor]
-| The text color functionality is now part of the {productname} core. For information on using and customizing the text color settings, see: link:{baseurl}/configure/content-appearance/#text_color[Content appearance options - `text_color`].
+| The text color functionality is now part of the {productname} core. For information on using and customizing the text color settings, see: xref:configure/content-appearance.adoc#text_color[Content appearance options - `text_color`].
```

Comment:
I see that in production that title has this id: `#textcoloroptions`, should we change `xref` to ```xref:configure/content-appearance.adoc#textcoloroptionsContent appearance options - `text_color`]``` instead?

Reply:
Yeah I think that makes sense, except we should do something like this instead since `text_color` isn't actually an option in 6.0:
```
xref:configure/content-appearance.adoc#textcoloroptions[Content appearance options - Text color options]
```

Reply:
done